docs/leverage.md

```diff
@@ -1,21 +1,80 @@
 # Leverage
+*You can't run 2 bots on the same account with leverage*
+
+!!! Warning "Trading with leverage is very risky"
+    *(`trading_mode="margin"` or `trading_mode="futures"`)* Do not trade with a leverage > 1 using a strategy that hasn't shown positive results in a live run using the spot market. Check the stoploss of your strategy. With a leverage of 2, a stoploss of 0.5 would be too low, and these trades would be liquidated before reaching that amount
+
+> I've only been using freqtrade for a couple weeks, but I feel like I'm pretty good and could use leverage
+
+**No you're not. Do not use leverage yet.**
+### Understand trading_mode
+
+The possible values are: `spot` (default), `margin`(*coming soon*) or `futures`.
+
+**SPOT**
+    Regular trading mode. 
+        - Shorting is not available
+        - There is no liquidation price
+        - Profits gained/lost are equal to the change in value of the assets(minus trading fees)
+
+#### Leverage trading modes
+
+# TODO: include a resource to help calculate stoplosses that are above the liquidation price
+
+#TODO: Taken from investopedia, is that ok?
+Leverage results from using borrowed capital as a funding source when investing to expand the firm's asset base and generate returns on risk capital. Leverage is an investment strategy of using borrowed money—specifically, the use of various financial instruments or borrowed capital—to increase the potential return of an investment. 
+
+
+**MARGIN**
+*coming soon*
+    Trading occurs on the spot market, but the exchange lends currency to you in an amount equal to the chosen leverage. You pay the amount lent to you back to the exchange with interest, and your profits/losses are multiplied by the leverage specified
+    
+**FUTURES**
+*Freqtrade can only trade **perpetual futures***
+
+    Perpetual futures contracts are traded at a price that mirrors the underlying asset they are based off of. You are not trading the actual asset but instead are trading a derivative contract. In contract to regular futures contracts, perpetual futures can last indefinately. 
+
+    In addition to the gains/losses from the change in price of the futures contract, traders also exchange funding fees, which are gains/losses worth an amount that is derived from the difference in price between the futures contract and the underlying asset. The difference in price between a futures contract and the underlying asset varies between exchanges.
+
+
+``` python
+"trading_mode": "futures"
+```
+
+### Collateral
+
+The possible values are: `isolated`, or `cross`(*coming soon*)
+
+# TODO: I took this definition from bitmex, is that fine? https://www.bitmex.com/app/isolatedMargin
```

Comment:
taking inspiration is for sure fine...
I think we'll need to add some of "our own" text to it though ... like "freqtrade will place a stoploss above the margin level to avoid forced liquidations" ... (and/or whatever else applies ...)